### PR TITLE
Conge

### PR DESCRIPTION
**Pull request by Nour-rouis**
Congé-Enseignant

### CODE REVIEW

File: src/pages/conge/enseignant/DemandeConge.jsx [hunk 1]
```diff
@@ -1,5 +1,155 @@
...
+            commentaire: autreChecked ? commentaire : "",
+            certificat: typeConge === "maladie" ? certificat : null,
+        })
+        navigateTo("/conge/enseignant/mes-demandes")
+    }
+
+    return (
+        <div style={{ display: "flex", justifyContent: "center" }}>
+            <form onSubmit={handleSubmit} style={{ width: "50%" }}>
+                <h1 style={{ textAlign: "center", fontWeight: "bold", fontSize: "70px", margin: "40px 0" }}>Demande Congé</h1>
```
Comment (stormynight9) [inline src/pages/conge/enseignant/DemandeConge.jsx:47]: esta3mel typography elli mawjouda lena:
https://ui.shadcn.com/docs/primitives/typography

File: src/pages/conge/enseignant/DemandeConge.jsx [hunk 2]
```diff
@@ -1,5 +1,155 @@
...
+                <div style={{ display: "flex", flexDirection: "column", gap: "16px" }}>
+                    <div>
+                        <Label htmlFor="dateDepart" style={{ fontSize: "20px", fontWeight: "bold" }}>
+                            Date de départ
+                        </Label>
+                        <Input type="date" id="dateDepart" style={{ fontSize: "20px" }} />
+                    </div>
```
Comment (stormynight9) [inline src/pages/conge/enseignant/DemandeConge.jsx:54]: esta3mel el label w input el7adhra mawjouda lena:
https://ui.shadcn.com/docs/primitives/input

File: src/pages/conge/enseignant/DemandeConge.jsx [hunk 3]
```diff
@@ -1,5 +1,155 @@
...
+                    <div>
+                        <Label htmlFor="dateRetour" style={{ fontSize: "20px", fontWeight: "bold" }}>
+                            Date de retour
+                        </Label>
+                        <Input type="date" id="dateRetour" style={{ fontSize: "20px" }} />
+                    </div>
```
Comment (stormynight9) [inline src/pages/conge/enseignant/DemandeConge.jsx:60]: esta3mel el label w input el7adhra mawjouda lena:
https://ui.shadcn.com/docs/primitives/input

File: src/pages/conge/enseignant/DemandeConge.jsx
Comment (stormynight9) [inline src/pages/conge/enseignant/DemandeConge.jsx:1]: esta3mel radio group elli mawjoud lena:
https://ui.shadcn.com/docs/primitives/radio-group

w nans7ek zid validation 3la 5ater najem nab3eth form far8a

File: src/pages/conge/enseignant/DemandeConge.jsx [hunk 4]
```diff
@@ -1,5 +1,155 @@
...
+                            <Input type="text" id="commentaire" value={commentaire} onChange={handleCommentaireChange} style={{ fontSize: "16px" }} />
+                            <label style={{ display: "flex", alignItems: "center", gap: "8px", marginTop: "8px" }}>
+                                <input type="checkbox" checked={autreChecked} onChange={() => setAutreChecked(!autreChecked)} />
+                                Je confirme que le commentaire ci-dessus est exact
+                            </label>
+                        </div>
+                    )}
+
+                    <div style={{ display: "flex", justifyContent: "center" }}>
+                        <Button type="submit" style={{ fontSize: "20px", fontWeight: "bold", marginTop: "32px" }}>
```
Comment (stormynight9) [inline src/pages/conge/enseignant/DemandeConge.jsx:146]: na7i styles mta3 button besh nes elkol ykounou kif kif

File: src/pages/conge/enseignant/MesDemandes.jsx
Comment (stormynight9) [inline src/pages/conge/enseignant/MesDemandes.jsx:1]: fel file hethi fama linting warning na7ih w zid ki n7el page yjini error fel console

File: src/pages/conge/enseignant/MesDemandes.jsx [hunk 5]
```diff
@@ -1,5 +1,133 @@
...
+                        <div className="flex items-center justify-center">
+                            {etat === "en cours" && (
+                                <DropdownMenu>
+                                    <DropdownMenuTrigger>
+                                        <button className="flex items-center justify-center">
+                                            <svg viewBox="0 0 24 24" width="16" height="16" stroke="currentColor" strokeWidth="2" fill="none" strokeLinecap="round" strokeLinejoin="round" className="text-gray-500">
+                                                <circle cx="12" cy="12" r="1"></circle>
+                                                <circle cx="6" cy="12" r="1"></circle>
+                                                <circle cx="18" cy="12" r="1"></circle>
+                                            </svg>
```
Comment (stormynight9) [inline src/pages/conge/enseignant/MesDemandes.jsx:69]: esta3mel icons elli hadhrin

File: src/pages/conge/enseignant/MesDemandes.jsx [hunk 6]
```diff
@@ -1,5 +1,133 @@
...
+                            )}
+                            {(etat === "accepté" || etat === "refusé") && (
+                                <DropdownMenu>
+                                    <DropdownMenuTrigger>
+                                        <button className="flex items-center justify-center">
+                                            <svg viewBox="0 0 24 24" width="16" height="16" stroke="currentColor" strokeWidth="2" fill="none" strokeLinecap="round" strokeLinejoin="round" className="text-gray-500">
+                                                <circle cx="12" cy="12" r="1"></circle>
+                                                <circle cx="6" cy="12" r="1"></circle>
+                                                <circle cx="18" cy="12" r="1"></circle>
+                                            </svg>
```
Comment (stormynight9) [inline src/pages/conge/enseignant/MesDemandes.jsx:86]: lena kif kif

File: src/pages/conge/enseignant/MesDemandes.jsx [hunk 7]
```diff
@@ -1,5 +1,133 @@
...
+                                        </button>
+                                    </DropdownMenuTrigger>
+                                    <DropdownMenuContent>
+                                        <DropdownMenuItem onClick={() => window.history.back()}>Modifier la demande</DropdownMenuItem>
```
Comment (stormynight9) [inline src/pages/conge/enseignant/MesDemandes.jsx:73]: nans7ek esta3mel useNavigate lel path elli hachtek bih 5ater method mta3ek traj3ek el e5er page kont feha w e5er page mch dima el form mta3 demade congé

File: src/pages/conge/enseignant/MesDemandes.jsx [hunk 8]
```diff
@@ -1,5 +1,133 @@
...
+                                        <button className="flex items-center justify-center">
+                                            <svg viewBox="0 0 24 24" width="16" height="16" stroke="currentColor" strokeWidth="2" fill="none" strokeLinecap="round" strokeLinejoin="round" className="text-gray-500">
+                                                <circle cx="12" cy="12" r="1"></circle>
+                                                <circle cx="6" cy="12" r="1"></circle>
+                                                <circle cx="18" cy="12" r="1"></circle>
+                                            </svg>
+                                        </button>
+                                    </DropdownMenuTrigger>
+                                    <DropdownMenuContent>
+                                        <DropdownMenuItem onClick={() => window.history.back()}>Envoyez une autre demande</DropdownMenuItem>
```
Comment (stormynight9) [inline src/pages/conge/enseignant/MesDemandes.jsx:90]: lena kif kif

**Review by molkaguesmi — COMMENTED** [review]
conge

**Pull request closed without merging**